frontend/src/pages/profile/index.tsx

```diff
@@ -107,6 +116,11 @@ const ProfilePage: NextPage = () => {
               <Grid item md={6} className={classes.card}>
                 <Orders data-test-id={`${ID_PREFIX}orders-`} />
               </Grid>
+              <Grid item md={6} className={classes.card}>
+                <PermissionRequired permission="cabins.add_booking">
```

Comment:
Technically, `add_booking` is the permission for creating new bookings, not for viewing/administering bookings. I suggest either requiring `change_booking`, or ideally add a new permission called `manage_bookings` or something of that sort to avoid any misinterpretations.

Reply:
True! Replacing `view_booking` with a new permission called `manage_booking`, used for all admin-related actions.